### ISSUE TITLE
Plugin approval

### ISSUE THREAD

**Issue by NyakudyaA** [FFouriauxEveha/ShapeOrientation#1]
How different is the plugin from the 
Line direction histogram Plugin ?

**Comment by FFouriaux** [comment]
Hello,

Thank you for the question and the time that you accord to this.

The Line direction histogram plugin decompose polygons for compute the direction of each  edges of polygons like lines. But in some cases, mine is the study of graves in archaeology, it is not significant. So we have to compute the major direction of the polygon entirely. That's what does ShapeOrientation.


You can see a example in the attachments the same polygon and two results with my plugin and the Direction Histogram plugin. The two results are corrects but don't lead to the same interpretation.

Sincerely

![testlinedirection](https://user-images.githubusercontent.com/18700472/31329983-cb8ef75e-acdc-11e7-96ba-f8fb30e5b651.png)
![testshapeorientation](https://user-images.githubusercontent.com/18700472/31329984-cb9f647c-acdc-11e7-9674-a584a0f2f67f.png)

**Comment by NyakudyaA** [comment]
If the two results would be the same but differ in interpretation does it not make sense to merge your work with the `Line direction histogram Plugin` and provide your method as an option so that a user can choose which algorithm to use ?.

**Comment by FFouriaux** [comment]
Yes, it could be. I wanted to make a very simple plugin cause there is already a lot of options in LineDirection, but it's only my personal point of view. 
I will contact the author of Line direction to see if it's possible.

**Comment by NyakudyaA** [comment]
Yes, I see they are plenty of options in Line direction and your plugin is simple and might be relatively easy to use. But we encourage people with plugins that do similar work to combine them so that users can have a better experience. If the functionality is different from the line orientation plugin I would be glad to approve the plugin. Can I open a ticket in Line direction plugin and hear what they say?

**Comment by FFouriaux** [comment]
I agree, it's better if the user don't have to install a lot of plugin when fonctions can be grouped in one. Ho yes, of course, if you could send a ticket it will be great.

**Comment by NyakudyaA** [comment]
have approved your plugin and once you can colloborate with the other plugin I can then remove it

**Comment by FFouriaux** [comment]
OK, thanks a lot. I hope we can integrate this function in Line direction. 
Sincerely
